src/current/v23.2/physical-cluster-replication-monitoring.md

```diff
@@ -0,0 +1,131 @@
+(1 row)
+~~~
+
+### Responses
+
+Field    | Response
+---------+----------
+`id` | The ID of the virtual cluster.
+`name` | The name of the standby (destination) virtual cluster.
+`data_state` | The state of the data on the virtual cluster. This can show one of the following: `initializing replication`, `ready`, `replicating`, `replication paused`, `replication pending cutover`, `replication cutting over`, `replication error`. Refer to [Data state](#data-state) for more detail on each response.
+`service_mode` | The service mode shows whether the virtual cluster is ready to accept SQL requests. This can show one of the following: <br>`none`<br>`shared`: The virtual cluster's SQL connections will be served by the same nodes that are serving the system interface.
+`source_tenant_name` | The name of the primary (source) virtual cluster.
+`source_cluster_uri` | The URI of the primary (source) cluster. This is the URI that connects to the primary cluster to [start a replication stream]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#step-4-start-replication).
+`replication_job_id` | The ID of the replication job.
+`replicated_time` | The latest timestamp at which the standby cluser has consistent data — that is, the latest time you can cut over to. This time advances while the replication proceeds without error. `replicated_time` is updated periodically (every `30s`).
+`retained_time` | The earliest timestamp at which the standby cluster has consistent data — that is, the earliest time you can cut over to.
+`cutover_time` | The time at which the cutover will begin.
+
+{% comment %}Add responses to include for SQL statement page{% endcomment %}
+
+#### Data state
+
+State      | Description
+-----------+----------------
+`initializing replication` | The replication job is completing the initial scan of data from the primary cluster before it starts replicating data in real time.
+`ready` | The virtual cluster's data is ready for use.
+`replicating` | The replication job has started and is replicating data.
+`replication paused` | The replication job is paused due to an error or a manual request with `ALTER VIRTUAL CLUSTER ... PAUSE REPLICATION`.
+`replication pending cutover` | The replication job is running and a cutover time has been set. Once the the replication reaches the cutover time, the will begin automatically.
+`replication cutting over` | The job has started cutting over. The cutover time can no longer be changed. Once complete, the virtual cluster will be available for use with `ALTER VIRTUAL CLUSTER ... START SHARED SERVICE`.
+`replication error` | An error has occurred. You can find more detail in the error message and the logs.
+
+## DB Console
+
+You can access the [DB Console]({% link {{ page.version.version }}/ui-overview.md %}) for your standby cluster at `https://{your IP or hostname}:8080/`. The **Physical Replication** dashboard under **Metrics** on the left-hand navigation bar.
```

Comment:
I had to get to this a different way. I had to go to Dashboards, and then select Physical Cluster Replication (not Replication) from the drop-down.

Reply:
Hmmmm, I think that is the same way as me... I have updated to make this clearer.